src/hooks/useVenueScheduleEvents.ts

```diff
@@ -104,7 +104,7 @@ const useVenueScheduleEvents = ({
   );
 
   // +1 to include the latest day in the schedule (for example, there are events tomorrow and today -> tomorrow - today + 1 = 2 days)
-  const dayDifference = daysInBetween + 1;
+  const dayDifference = daysInBetween + 1 || 0;
```

Comment:
[NOTE] Not really necessary to change this code, maybe just add a line of comment saying `undefined + 1 is NaN which is falsy`.

On other hand, depending on `NaN` being falsy might not be obvious to everyone why it works.Can it be something like `isDefined()` check or `?? 0` ? Just a question, would keep the logic as is.

Reply:
added a comment + converted to ternary [80e0fc3](https://github.com/sparkletown/sparkle/pull/2955/commits/80e0fc3d5a99f391b995b7e6baf0a44ba5a0ec0f)